direct/src/interval/FunctionInterval.py

```diff
@@ -86,11 +86,18 @@ def makeUniqueName(func, suffix = ''):
         return name
 
     def privInstant(self):
+        """
+        In versions previous to 1.11, this function did not set the state of the Interval
+        In 1.11 this has been changed to be consistent with MetaInterval
+        """
```

Comment:
IMO for the purposes of generating the API documentation it's not great to have a docstring on a private method, I'd be OK with omitting this or turning it into a regular comment